packages/fxa-auth-server/test/oauth/api.js

```diff
@@ -2849,38 +2852,7 @@ describe('/v1', function() {
         .then(function(res) {
           assert.equal(res.statusCode, 400);
           assertSecurityHeaders(res);
-          assert.equal(res.result.errno, 115);
-        })
-        .finally(function() {
-          config.set('oauthServer.expiration.accessTokenExpiryEpoch', epoch);
-        });
-    });
-
-    it('should accept expired tokens from before the epoch', function() {
```

Comment:
There's a shift in behavior here since redis tokens are actually deleted when they expire so there's no "token exists but is expired" state (except for pocket tokens which are still in mysql)

I think it makes sense to return a 108 "Invalid token" here.